introduction/CAPMC_reinit_and_config.md

```diff
@@ -0,0 +1,78 @@
+# CAPMC Reinit and Configuration Notice
+
+CAPMC is now capable of doing a reinit of hardware that does not support
```

Comment:
Spell out the acronym on first use.

The Cray Advanced Platform Monitoring and Control (CAPMC)...